api/c/indigo/src/indigo.cpp

```diff
@@ -41,6 +42,15 @@
 
 static _SessionLocalContainer<Indigo> indigo_self;
 
+static const std::string kVersionInfo = R"({
```

Comment:
Not sure about the place where we need to store this data

Reply:
please find where the version is created. The string is populated automatically - but not static

Reply:
Updated. Current output result is number of version: 1.14.0